config/locales/en.yml

```diff
@@ -5184,13 +5187,18 @@ en:
             service_description: File for multiple years with an IRS-certified tax preparer.
             service_name: GetYourRefund
             service_name_short: GYR
-        state_credits: Additional State Tax Credits
         still_need_help: Still need help selecting the best service?
-        stimulus_payments: Stimulus Payments
-        support_via_gyr: available through GetYourRefund
         tax_documents:
           subtitle: "(w2s, 1099s, etc)"
           title: Official tax documents
+        tax_payer_residency:
+          direct_file: Available to full-year residents of Arizona, California, Florida, Massachusetts, New Hampshire, New York, Nevada, South Dakota, Tennessee, Texas, Washington, Wyoming.
```

Comment:
random nit: is there supposed to be an "and" before Wyoming?

Reply:
there was no and in the copy lol but should i add it?